string image_name
---
bool success
string image_path